### ISSUE TITLE
easy way to add -h and -? help options

### ISSUE THREAD

**Issue by hsn10** [scallop/scallop#73]
Is there some easy way to add -h and -? as aliases to builtin --help option?

**Comment by hsn10** [comment]
is there way to make -? option without making --long one?

**Comment by Rogach** [comment]
No, there's no way to create short-name-only option currently :(
In your situation, you can avoid that via specifying your own help option:

``` scala
def help = opt("help", '?', descr = "show help message")
```

That won't work for two short names, though.

**Comment by hsn10** [comment]
this have one disadvantage. 

I have some options marked as required. If i will do user defined help as you stated then if i invoke -? i will get error message about required option missing.

**Comment by hsn10** [comment]
``` scala
val help = opt[Boolean] (name = "help", short = '?', descr = "print help")
```

--help works as expected
-? complains about required option missing

**Comment by Rogach** [comment]
Ah yes, I see now. That's surely a bug. I'll look into it later today.

About several short names for the same option - that's a bit trickier, I will not be able to add that feature right now.

**Comment by Rogach** [comment]
I fixed the bug, and published a snapshot (for scala 2.10.x):

``` scala
resolvers += "sonatype snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
libraryDependencies += "org.rogach" %% "scallop" % "0.9.3~3~SNAPSHOT"
```
